### PR TITLE
feat: update docs

### PR DESCRIPTION
**Pull request by mswertz**
please only shallow review to check for major confusion. While this is work in progress, I believe it is good to get it into view.

To review
[x] look at README.md
[x] cd into docs and run `docsify serve` 

N.B. if you didn't install before also install docsify using `npm i docsify-cli -g`

### CODE REVIEW

File: docs/dev_quickstart.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+IntelliJ.
+
+## Build whole system
+
+Requires [Postgresql 13](https://www.postgresql.org/download/) and java (we use
+adopt [OpenJDK 16](https://adoptopenjdk.net/)):
+
+* Optionally, drop a previous version of molgenis database (caution: destroys previous data!)
+  ```console
+  sudo -u postgres psql
```
Comment (sidohaakma) [inline docs/dev_quickstart.md:30]: This is only relevant for Mac or Linux users. Or can you do this in powershell as well? I would imagine in Windows you would simply use the postgres app. I would suggest using docker serving the postgres backend. You can simply state `docker-compose up postgres` then you have a way of working agnostic to all platforms.

File: docs/dev_quickstart.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  postgres=# drop database molgenis;
+  ```
+* If not already done, create postgresql database with name 'molgenis' and with superadmin user/pass 'molgenis'. On
+  Linux/Mac commandline:
+  ```console
+  sudo -u postgres psql
```
Comment (sidohaakma) [inline docs/dev_quickstart.md:36]: See previous comment.

File: docs/dev_technologies.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+We also happy users of  [graphql-java](https://github.com/graphql-java/graphql-java) graphql+java
+implementation, [Jackson](https://github.com/FasterXML/jackson) data processing
+suite, [Vue Styleguidist](https://vue-styleguidist.github.io/) frontend component development environment with living
+styleguide, [rdf4j](https://rdf4j.org/)
+modular Java framework for RDF data, [HikariCP](https://github.com/brettwooldridge/HikariCP) JDBC connection pool,
+log4j, junit, javers, font-awesome, vue-swatches, vuedraggable, v-scroll-lock, ckeditor4-vue, vue-flatpickr-component,
+vue-runtime-template-compiler.
+
+##
```
Comment (sidohaakma) [inline docs/dev_technologies.md:26]: What is this?

File: docs/use_database.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Creating/managing databases
```
Comment (sidohaakma) [inline docs/use_database.md:1]: Leave it out and create a story to develop the missing parts.

File: docs/use_forms.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# Data capture forms
```
Comment (sidohaakma) [inline docs/use_forms.md:1]: Leave it out and create a story to develop the missing parts.

File: docs/use_tables.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Table explorer
```
Comment (sidohaakma) [inline docs/use_tables.md:1]: Leave it out and create a story to develop the missing parts.

File: docs/use_updownload.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# Up and download tool
```
Comment (sidohaakma) [inline docs/use_updownload.md:1]: Leave it out and create a story to develop the missing parts.

**Review by sidohaakma — CHANGES_REQUESTED** [review]

File: docs/dev_quickstart.md (hunk 1 above)
Comment (mswertz) [reply docs/dev_quickstart.md:30]: I think this manual is meant for developers and those using Linux. If you are not able to do this, you shouldn't be a developer of the backend in my mind. We have docker compose for other users, incl frontend developers, If you are a windows user (I was one) I think you should figure this out yourself. 

So my proposed action is to make specific that this is for Linux/Mac users and that Windows users should tell us what needs to be different (I could ask Bart).

**Comment by sidohaakma** [comment]
[Pull request 'docs'](https://trello.com/c/lb4c3KFC/156-pull-request-docs)

File: docs/dev_quickstart.md (hunk 2 above)
Comment (mswertz) [reply docs/dev_quickstart.md:36]: Se previous reply

File: docs/use_forms.md (hunk 5 above)
Comment (mswertz) [reply docs/use_forms.md:1]: I rather keep it, I will add a short summary on where the forms live.

File: docs/use_updownload.md (hunk 7 above)
Comment (mswertz) [reply docs/use_updownload.md:1]: I added rudimentary info.

File: docs/dev_technologies.md (hunk 3 above)
Comment (mswertz) [reply docs/dev_technologies.md:26]: will add header.

File: docs/use_tables.md (hunk 6 above)
Comment (mswertz) [reply docs/use_tables.md:1]: added a few more contents

File: docs/use_database.md (hunk 4 above)
Comment (mswertz) [reply docs/use_database.md:1]: added basic info here too.

**Comment by mswertz** [comment]
I have added minimum information to get rid of 'todo'. Of course it is minimal but I think still useful.

**Pull request merged**